README.md

```diff
@@ -19,6 +19,34 @@ Once the website finishes launching the website will be available on port 80: [h
 Part of the deployment builds the docker containers used by the website, so it may take a few minutes to finish deployment.
 To stop the app run `docker-compose down`.
 
+### Certbot Setup on AWS
+Certificates can be installed to the EC2 VM using Certbot with Nginx on pip following [EFF instructions](https://certbot.eff.org/instructions?ws=nginx&os=pip).
+```bash
+sudo python3 -m venv /opt/certbot/
+sudo /opt/certbot/bin/pip install --upgrade pip
+sudo /opt/certbot/bin/pip install certbot certbot-nginx
+sudo ln -s /opt/certbot/bin/certbot /usr/bin/certbot
+```
+And to install Nginx on the Amazon Linux 2023 (following [instructions](https://awswithatiq.com/how-to-install-nginx-in-amazon-linux-2023/)) and run Certbot:
+```bash
+docker-compose down
+sudo dnf update -y
+sudo dnf install nginx -y
+sudo systemctl start nginx
+sudo certbot certonly --nginx
+```
+Following prompted agreements, registration, and specifying andromeda.imageomics.org, the certificates are installed to:
+
+Certificate: `/etc/letsencrypt/live/andromeda.imageomics.org/fullchain.pem`
+
+Key: `/etc/letsencrypt/live/andromeda.imageomics.org/privkey.pem`
+
+Finally, shut down Nginx on the system to avoid interference with the container:
+```bash
+sudo systemctl stop nginx
+nohup docker-compose up -d &
```

Comment:
Docker compose runs everything in the background when run with `-d` the `nohup` and `&` part are not necessary.
```suggestion
docker-compose up -d
```

Reply:
I think the benefit of `nohup` is logging to `nohup.out`. Is the enough to warrant keeping it?

Reply:
`docker-compose up -d` returns immediately after starting docker containers in the background - see https://docs.docker.com/engine/reference/commandline/compose_up/. `nohup` will keep a process running even if the terminal that launched it closes. This process doesn't keep running so that isn't needed. The only output is the status of the containers it started. I would rather see this information in the terminal when running the command rather than a log file. If you wanted to save output to a log file there are simpler tools than `nohup`, but I don't see the need in this case.

Reply:
OK, that makes sense. Change applied!